docs/_basic/authenticating_oauth.md

```diff
@@ -24,20 +62,18 @@ const app = new App({
   clientSecret: process.env.SLACK_CLIENT_SECRET,
   stateSecret: 'my-state-secret',
   scopes: ['chat:write'],
-  // optional code for directInstall. Defaults to false. When set to true, user does not see "Add to Slack" button page. 
```

Comment:
I felt that including this additional code block was adding additional bloat to an already pretty big section. I've added a git permalink instead that points to the lines in the example oauth app where directInstall option can be seen clearly.

Reply:
Makes sense!